.travis.yml

```diff
@@ -15,7 +15,7 @@ install:
   - pip install dist/feedgen*
 
 script:
-  - make test
+  # - make test
```

Comment:
https://github.com/pitpa/python-feedgen/pull/1/files#r813595205

Reply:
I readded it into this file (I remove it before just for testing)